tests/test_l3_interface.py

```diff
@@ -23,20 +23,20 @@
 except ImportError:  # py2
     import mock
 
-import sys
+mock_modules = {}
```

Comment:
You could create a dict in some common place and return it directly in line 37.
Not sure about the __import__ portion: Does it matter where it is called?

Reply:
Yes, I thought about this as well... The import is different in each file.

Reply:
So can you then do something like:
mock_modules(__import__("ansible_nmstate")) ?

Reply:
ah, I meant the import that is patched... this one could be part of the common code.